package.json

```diff
@@ -83,22 +86,30 @@
     "@types/mime-types": "^2.1.0",
     "@types/n3": "^1.4.4",
     "@types/node": "^14.10.2",
-    "@types/rdf-js": "^4.0.0",
     "@types/sparqljs": "^3.1.0",
     "@types/streamify-array": "^1.0.0",
     "@types/uuid": "^8.3.0",
     "@types/ws": "^7.4.0",
     "@types/yargs": "^16.0.0",
     "arrayify-stream": "^1.0.0",
+    "assert": "^2.0.0",
     "async-lock": "^1.2.4",
+    "bcrypt": "^5.0.0",
+    "clownface": "^1.2.0",
```

Comment:
I'd like to remove this dependency; it's too niche.

Reply:
😞

Reply:
Removed